docs/pages/guides/mesheryctl/authenticate-with-meshery-via-cli.md

```diff
@@ -11,27 +11,26 @@ To authenticate with Meshery through `mesheryctl` you will use the command `mesh
 
 ## Get your Token
 
-You can retrieve your authentication token from either of Meshery's two clients: the CLI or the UI. 
+You can retrieve your authentication token from either of Meshery's two clients: the CLI or the UI.
 
 - Get your token through [Meshery UI](/concepts/architecture/ui), from the `Get Token` option.
 
-    _Downloading the token_
+  _Downloading the token_
 
-    <a href="{{ site.baseurl }}/assets/img/token/token.png"><img alt="Meshery Dashboard" src="{{ site.baseurl }}/assets/img/token/token.png" /></a>
-    <br/>
-    <br/>
+  <a href="{{ site.baseurl }}/assets/img/token/token.png"><img alt="Meshery Dashboard" src="{{ site.baseurl }}/assets/img/token/token.png" /></a>
+  <br/>
+  <br/>
 
 - Get your token through **Meshery CLI**.
-    <br/>
-   > To get the token through `mesheryctl` you would have to use the following command and the path to token for authenticating to Meshery API (default "auth.json").
-    <br/>
-    <pre class="codeblock-pre">
-        <div class="codeblock">
-        mesheryctl system config --token [path-to-file]
-        </div>
-    </pre>
-<br />
-
+  <br/>
+  > To get the token through `mesheryctl` you would have to use the following command and the path to token for authenticating to Meshery API (default "auth.json").
+      <br/>
+      <pre class="codeblock-pre">
```

Comment:
hey @shubham-tyagi could you please check what's happening in this part, this is supposed to be a code block, but it's appearing in form of normal text.

Reply:
thanks for reviewing @warunicorn19.  Are there any same issues anywhere else or just this?

Reply:
I couldn't find anymore 😅

Reply:
fixed the codeblock issue @warunicorn19